lms/static/js/learner_dashboard/views/program_header_view_2017.js

```diff
@@ -0,0 +1,76 @@
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+        'jquery',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'text!../../../templates/learner_dashboard/program_header_view_2017.underscore',
+        'picturefill',
+        'text!/static/images/programs/micromasters-program-details.svg',
```

Comment:
@AlasdairSwan if i comment out these images my program_details_view_spec.js test work  
otherwise the tests error on a 404 on these. what is the correct way to include them in the spec file?